msal/build.gradle

```diff
@@ -5,6 +5,16 @@ apply plugin: 'maven'
 apply plugin: 'maven-publish'
 apply from: 'versioning/version_tasks.gradle'
 group = 'com.microsoft.identity.client'
+apply plugin: 'jacoco'
+
+jacoco {
+    reportsDir = file("$buildDir/reports/unitTests")
+}
+
+//To include Robolectric tests in the Jacoco report, flag -> "includeNolocationClasses" is set to true
```

Comment:
Can you provide more details as to why this is needed?

Reply:
So whenever this flag is not set to true or enabled. there is no coverage which happens for Roboelectric tests and as msal and other repos contains these Roboelectric tests. so we have to enable this class. And in earlier versions of jacoco, this flag was already enabled in jacoco plugin itself but in latest versions it is not there so we have to do it manually.